dns_reverse_proxy.go

```diff
@@ -82,7 +82,7 @@ func main() {
 			}
 			backends = append(backends, backend)
 		}
-		if !strings.HasSuffix(s[0], ".") {
+		if !strings.HasSuffix(strings.ToLower(s[0]), ".") {
```

Comment:
Is it really necessary here? we're checking for "." which does not have uppercase or lowercase.

Reply:
Erm, no. This is the wrong place to add it. It should be on line 88 so as to save the routes into the map in lower case. I'll fix this on Tuesday.